monkestation/code/modules/mob/living/carbon/human/species_type/ipc.dm

```diff
@@ -155,10 +155,12 @@
  * * screen_name - The name of the screen to switch the ipc_screen mutant bodypart to. Defaults to BSOD.
  */
 /datum/species/ipc/proc/bsod_death(mob/living/carbon/human/transformer, screen_name = "BSOD")
-	saved_screen = change_screen // remember the old screen in case of revival
-	switch_to_screen(transformer, screen_name)
-	addtimer(CALLBACK(src, PROC_REF(switch_to_screen), transformer, "Blank"), 5 SECONDS)
-
+	if(!transformer.get_bodypart(BODY_ZONE_HEAD))
+		return
+	else
+		saved_screen = change_screen // remember the old screen in case of revival
+		switch_to_screen(transformer, screen_name)
+		addtimer(CALLBACK(src, PROC_REF(switch_to_screen), transformer, "Blank"), 5 SECONDS)
```

Comment:
```suggestion
	if(!transformer.get_bodypart(BODY_ZONE_HEAD))
		return
	saved_screen = change_screen // remember the old screen in case of revival
	switch_to_screen(transformer, screen_name)
	addtimer(CALLBACK(src, PROC_REF(switch_to_screen), transformer, "Blank"), 5 SECONDS)
```